app/editor/src/components/wysiwyg/Wysiwyg.tsx

```diff
@@ -90,9 +90,20 @@ export const Wysiwyg = <T extends object>({
   const stripHtml = () => {
     // strip html from string
     if (!!fieldName) {
-      let doc = new DOMParser().parseFromString((values[fieldName] as string) ?? '', 'text/html');
-      setFieldValue(fieldName as string, doc.body.textContent || '');
-      setState({ ...state, html: doc.body.textContent || '' });
+      const doc = new DOMParser().parseFromString(
```

Comment:
Can you explain what is being done here?  It doesn't make sense to me.

Reply:
The changes here are basically adjusting the remove formatting tool to meet the expectation as mentioned in the ticket, and we can have a quick meeting to chat about it if needed. Thanks

Reply:
What I don't understand is why it's replacing all <p> and <br> temporarily with [p] and [br]?  Is it because the `parseFromString()` function would remove them if you didn't do this?

What about paragraph tags with attributes?

Reply:
I though the DOMParser().parseFromString() turns a string into HTML.  Why would it remove html elements?

Reply:
I think I kinda understand what's going on here.  You're using the DOMParser to get the `textContent` of the body element.  Just make the below change.

Reply:
@Fosol  : `parseFromString` is turning a string into HTML but then the lines below it used to just take the `.textContent` from that parsed string.  

@CelloLiu : instead of doing straight up replaces, you could try for regular expressions that target those tags. If you can do a regex which targets `<p *>` and replaces it with just the `[p]` token that might work.  Also remember that the `<br>` tag can also be written `<br/>` as it's a self closing tag.

Reply:
The DOMParser().parseFromString function won't remove the tags, however, as a result its body.textContent will lose these paragraph tags, so in order to preserve them we temporarily replace them all with the [p] and [br] and convert them back later to keep the formatting. 

For paragraph tags with attributes, good question, I thought of it before and we could deal with it too if needed, however, it looks like when we update the text in the editor the related attributes would be added/updated only to the sub section of the paragraph tag instead of adding the attributes to the paragraph tag directly, so that is why we are not considering this scenario yet for the moment. Thanks

Reply:
@kylgarmor: Currently when we update the text in the editor, the editor will use <br> directly instead of <br/> or <br />, and for the paragraph tag it will use <p> directly and add any attributes to the subsection of the p tag instead, but sure we could make the changes to take care of the potential attributes issue for this case as well. Thanks

Reply:
@CelloLiu Don't we have any ingest source from whom we get html markup already? If so, those tags could have any number of attributes on them.

Reply:
Updated, thanks